subsys/lorawan/lorawan.c

```diff
@@ -576,7 +578,7 @@ int lorawan_start(void)
 	return 0;
 }
 
-static int lorawan_init(const struct device *dev)
+int lorawan_init(void)
```

Comment:
As far as I can see, this function is not called anywhere. Did I missed something?

Reply:
No, it's my bad. LoRa is not my priority right now, and I've made the code changed without testing it.
If possible, test this PR as I really don't have the bandwidth right now to do it.

Reply:
Ok, no problem. Just wanted to know what the status is and how we can go ahead. I'll have a look at your PR next week and do more testing.